sdk/sql/mgmt/src/main/java/com/azure/management/sql/implementation/SqlDatabaseExportRequestImpl.java

```diff
@@ -13,19 +14,23 @@
 import com.azure.management.sql.SqlDatabaseExportRequest;
 import com.azure.management.sql.SqlDatabaseImportExportResponse;
 import com.azure.management.sql.StorageKeyType;
+import com.azure.management.storage.BlobContainer;
+import com.azure.management.storage.BlobContainers;
+import com.azure.management.storage.PublicAccess;
 import com.azure.management.storage.StorageAccount;
-import com.azure.storage.blob.BlobServiceClient;
-import com.azure.storage.blob.BlobServiceClientBuilder;
-import com.azure.storage.blob.implementation.models.StorageErrorException;
-import java.util.Objects;
-import reactor.core.Exceptions;
 import reactor.core.publisher.Mono;
 
+import java.util.Objects;
+
 /** Implementation for SqlDatabaseExportRequest. */
 public class SqlDatabaseExportRequestImpl extends ExecutableImpl<SqlDatabaseImportExportResponse>
     implements SqlDatabaseExportRequest, SqlDatabaseExportRequest.SqlDatabaseExportRequestDefinition {
 
     private final ClientLogger logger = new ClientLogger(getClass());
+    private static final String STORAGE_CONNECTION_STRING =
+        "DefaultEndpointsProtocol=https;" +
+            "AccountName=%s;" +
+            "AccountKey=%s";
```

Comment:
OK for now. One uncertainty is whether `EndpointSuffix` will be needed.

Reply:
Has removed since we don't use data-plane now